### PR TITLE
Responsive styles for Dashboard

### PR DESCRIPTION
**Pull request by WilkinsKa1**
## Description

As the rest of the responsive work has been handled in separate component tasks, this PR is mainly adding responsive styles/markup for the notification center. Comments are throughout to explain the details, but we are using Hidden to 'toggle' content. A new CMR-specific component has been created for styles and to support the 'item count' displayed on the mobile version notification items.

## Type of Change
- Non breaking change ('update')

### CODE REVIEW

File: packages/manager/src/features/Dashboard_CMR/DashboardNotifications.tsx [hunk 1]
```diff
@@ -88,29 +93,40 @@ export const Notifications: React.FC<{}> = _ => {
...
-            <Grid container direction="column">
-              <Grid item>
-                <PendingActions />
-              </Grid>
-              <Grid item>
-                <Alerts />
-              </Grid>
-              <Grid item>
-                <Maintenance />
+          <Hidden smDown>
```
Comment (WilkinsKa1) [inline packages/manager/src/features/Dashboard_CMR/DashboardNotifications.tsx:96]: Only change here is wrapping the content in `Hidden`

File: packages/manager/src/components/ExtendedExpansionPanel/ExtendedExpansionPanel.tsx [hunk 2]
```diff
@@ -56,12 +59,37 @@ const ExtendedExpansionPanel: React.FC<Props> = props => {
...
   return (
-    <ExpansionPanel heading={heading} onChange={onChange}>
-      {renderContent(error, Boolean(loading), height || 300, renderMainContent)}
-    </ExpansionPanel>
+    <React.Fragment>
+      {flags.cmr ? (
+        <ExpansionPanelCMR
+          heading={heading}
+          onChange={onChange}
+          headingNumberCount={headingNumberCount}
```
Comment (WilkinsKa1) [inline packages/manager/src/components/ExtendedExpansionPanel/ExtendedExpansionPanel.tsx:73]: How we are passing the item count for each individual notification section (and isolating style differences from non-CMR)

File: packages/manager/src/components/ExpansionPanel/ExpansionPanelCMR.tsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+import Typography, { TypographyProps } from 'src/components/core/Typography';
+import Grid from 'src/components/Grid';
+import RenderGuard from 'src/components/RenderGuard';
+import Notice from '../Notice';
+
+type ClassNames = 'root' | 'success' | 'warning' | 'error' | 'itemCount';
+
+const styles = (theme: Theme) => {
+  return createStyles({
+    // @todo: Post-CMR these overrides should live in the theme file.
```
Comment (WilkinsKa1) [inline packages/manager/src/components/ExpansionPanel/ExpansionPanelCMR.tsx:27]: This is gross but temporary- we do this elsewhere already but definitely a cleanup item when CMR takes over.

File: packages/manager/src/features/NotificationCenter/NotificationSection.tsx [hunk 4]
```diff
@@ -59,21 +61,10 @@ export type CombinedProps = Props;
...
-              <strong>
-                <Link to={showMoreTarget}>
-                  {showMoreText ?? 'View history'}
-                </Link>
-              </strong>
-            </Typography>
-          )}
-        </div>
+
+  const innerContent = () => {
```
Comment (Jskobos) [inline packages/manager/src/features/NotificationCenter/NotificationSection.tsx:65]: I ended up doing something similar in #6715. But different. One of us has an unpleasant rebase ahead of us 😆

Comment (WilkinsKa1) [reply packages/manager/src/features/NotificationCenter/NotificationSection.tsx:65]: Definitely plan on overriding the change here in favor of yours- mine is just whatever you had before so it was easier to add to the panel markup

**Comment by WilkinsKa1** [comment]
Thanks @Jskobos ! The blue was actually left over from MUI's styling, but we should probably have a differentiating focus/active state- I'll ask for clarification on that. Along the same line, I believe the blue is persisting because it is still the focused, 'active' element. We could update that so once you close the panel, focus/activated state is removed.

**Comment by WilkinsKa1** [comment]
Updated to rotate the arrow on expansion 👍

**Pull request merged**